plugins/module_utils/foreman_helper.py

```diff
@@ -441,6 +462,16 @@ def _patch_organization_update_api(self):
         _organization_update_params_organization = next(x for x in _organization_update['params'] if x['name'] == 'organization')
         _organization_update_params_organization['required'] = False
 
+    def _patch_subscription_index_api(self):
+        """This is a workaround for the broken subscriptions apidoc in katello.
```

Comment:
redmine link? :)

Reply:
Can we piggyback https://projects.theforeman.org/issues/27527 ?

Reply:
I'd prefer not to, so I've created https://projects.theforeman.org/issues/27575